ptx/sec_FTC.ptx

```diff
@@ -913,8 +1063,21 @@
         <figure xml:id="fig_ftc6">
           <caption>Sketching the region enclosed by <m>y=x^2+x-5</m> and <m>y=3x-2</m> in <xref ref="ex_ftc6">Example</xref></caption>
           <image xml:id="img_ftc6" width="47%">
-            <description></description>
-            <latex-image>
+            <shortdescription>
+              Graph of region enclosed by y=x^2+x-5 and y=3*x-2.
+              </shortdescription>
+              <description>
+              <p>
+              The <m>y</m> axis is drawn from <m>-5</m> to <m>15</m> and 
+              the <m>x</m> axis is drawn from <m>-2</m> to <m>4</m>. 
+              The first function <m>x^2 +x-5</m>is a curve that has a 
+              <m>y</m> intercept at <m>-5</m> and the <m>x</m> intercept 
+              near <m>1.8</m>. The second function <m>3*x-2</m> is a 
+              straight line. The two lines intersect at points <m>(-1,-5)</m> 
+              and <m>(3,7)</m>. The area bounded by the two functions is shaded.
+              </p>
+              </description>
+                          <latex-image>
```

Comment:
The `<latex-image>` tag keeps getting indented. The book source uses 2 space indentation:
```
<description>
  <p>
    The <m>y</m> axis...
  </p>
</description>
```
It's better if you can stick to this indentation scheme.